requirements.txt

```
@@ -8,6 +8,6 @@ starsessions==1.2.3
 numpy==1.24.3
 Pillow==9.5.0
 opencv-python-headless==4.7.0.72
-torch==2.0.0
+torch==2.0.1
 torchvision==0.15.1
```

```suggestion
torchvision==0.15.2
```
fuse together with #48